cybox/test/__init__.py

```diff
@@ -52,8 +54,8 @@ def round_trip(o, output=False, list_=False):
 
     klass = o.__class__
     if output:
-        print "Class: ", klass
-        print "-" * 40
+        print("Class: ", klass)
```

Comment:
Cool cool. Nose has a lot of [logging options](http://nose.readthedocs.org/en/latest/plugins/logcapture.html) and by default displays log messages on error.